cli/cmd/generate.go

```diff
@@ -0,0 +1,348 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/imdario/mergo"
+	"github.com/lacework/go-sdk/lwgenerate/aws"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type AwsGenerateCommandExtraState struct {
+	Output                string
+	UseExistingCloudtrail bool
+	AwsSubAccounts        string
+}
+
+var (
+	QuestionRunTfPlan            = "Run Terraform plan now?"
+	GenerateAwsCommandState      = &aws.GenerateAwsTfConfigurationArgs{}
+	GenerateAwsExistingRoleState = &aws.ExistingIamRoleDetails{}
+	GenerateAwsCommandExtraState = &AwsGenerateCommandExtraState{}
+
+	// iac-generate command is used to create IaC code for various environments
+	generateTfCommand = &cobra.Command{
+		Use:     "iac-generate",
+		Aliases: []string{"iac-generate", "iac"},
```

Comment:
I wonder if we don't need to specify `iac-generate` in the Aliases field?
When I run it locally I see it duplicated in the help output
![image](https://user-images.githubusercontent.com/15362434/145017323-0a545f8e-644d-4e0e-8b8b-7e21ce108b3f.png)

Reply:
Good catch; will update